charts/trow/values.yaml

```diff
@@ -20,6 +21,7 @@ trow:
     allowImages: []
     disallowLocalPrefixes: []
     disallowLocalImages: []
+    proxyDockerHub: true
```

Comment:
I think it should probably be false by default

Reply:
Ok, I changed it to false